scripts/functions

```diff
@@ -1305,6 +1305,24 @@ CT_DoBuildTargetTuple() {
     CT_ARCH_TARGET_LDFLAGS="${CT_ARCH_TARGET_LDFLAGS} ${CT_ARCH_ENDIAN_LDFLAG}"
 }
 
+# This function determines the target tuple for a given set of compiler
+# flags, using either GCC's multiarch feature (if supported) or using
+# fallback architecture-specific functions.
+CT_DoMultilibTarget() {
+    local multi_flags="$1"
+    local target="$2"
+    local gcc_multiarch
+
+    gcc_multiarch=$("${CT_TARGET}-gcc" -print-multiarch ${multi_flags})
```

Comment:
Re quoting: it is not necessary, since the expression on the right is a single bash token - `$(...)`. Any whitespace generated as a result of expansion is still retained, e.g.:
````
avn@mistral:~/work/ctng/crosstool-ng$ a="a   b "
avn@mistral:~/work/ctng/crosstool-ng$ b=${a}
avn@mistral:~/work/ctng/crosstool-ng$ echo "[$b]"
[a   b ]
````

If you insist, I'll do that - but I think unnecessary nested quoting just reduces readability.